packages/amplify_core/lib/src/types/analytics/hub/analytics_hub_event.dart

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_core/amplify_core.dart';
+
+/// {@template amplify_common.hub.analytics_hub_event_type}
+/// Hub Event types for the Analytics category.
+/// {@endtemplate}
+enum AnalyticsHubEventType {
+  /// {@macro amplify_common.hub.analytics_hub_event_pinpoint_endpoint_id_set}
+  pinpointEndpointIdSet('PINPOINT_ENDPOINT_ID_CONFIGURED');
```

Comment:
You've only updated the string value from "set" -> "configured". More importantly, though, I was suggesting something shorter and more generic like just `configured`, `endpointConfigured`, or `endpointUpdated`. Although, I think `endpointConfigured` here works best.

```suggestion
  endpointConfigured('ENDPOINT_CONFIGURED'),
```

Hub events should not mention AWS services since they are generic to the category. Further, the event that triggers this hub event is the successful creation of an "endpoint" which happens to include an ID. Since we already have a category API called `updateEndpoint`, piggy backing on that verbiage would seem to be the most appropriate.

An `endpointUpdated` event could also make sense which is called after each `updateEndpoint` call and includes the full endpoint instead of just the ID.

Reply:
Right thanks for clarifying that.  I think it would be good to mention endpoint to specify the exact nature of the configuration here.